src/core/lwt_stream.mli

```diff
@@ -45,10 +45,8 @@ val create : unit -> 'a t * ('a option -> unit)
 (** [create ()] returns a new stream and a push function.
 
     To notify the stream's consumer of errors, either use a separate
-    communication channel, or use a
-    {{:https://ocaml.org/api/Stdlib.html#TYPEresult}
-    [result]} stream. There is no way to push an exception into a
-    push-stream. *)
+    communication channel, or use a {!Stdlib.result} stream. There is
```

Comment:
I've had issues with this in some context. Specifically, with links to outer dependencies. Is this expected to works with some specific versions of odoc? With some specific environment? With some specific options?

I just tested locally to run `dune build -p lwt @doc` and I get a lot of warnings including:

```
File "src/core/lwt_stream.mli", line 48, characters 36-52:
Warning: Failed to resolve reference unresolvedroot(Stdlib).result Couldn't find "Stdlib"
```

Reply:
That's due to Dune not building the documentation of the dependencies. The documentation can't contain any link to the Stdlib's doc, because it is not built.

I just tried using Odig, which builds the documentation for the stdlib, and I confirm this reference resolves correctly.